OpenOversight/app/static/css/openoversight.css

```diff
@@ -568,3 +568,8 @@ tr:hover .row-actions {
 .bottom-10 {
     bottom: 10px;
 }
+
+.officer-face {
+    min-width: 200px;
+    min-height: 200px;
+}
```

Comment:
I'm not familiar with OO, are all pictures set to 1:1 aspect ratio?
If not, setting `min-width` and `max-height` (i.e. change `min-height` to `max-height`) to the same value should handle this properly across all aspect ratios. That would make it less wide for taller pictures tho.

Reply:
No, not all pictures are 1:1. My intent was, regardless of the aspect ratio, images should be _at least_ 200px to a side. I think OO might already have a max size for images, but I wouldn't want to define a max height for photos that was too small. If you have a picture that's 100x400, which rule would be respected: min width of 200px or max height of 200px?